### DISCUSSION TITLE
Donate

### DISCUSSION THREAD

**Discussion by eddvys** [debauchee/barrier#1269] [Q&A]
I am extremely sorry if this is already somewhere within doc's but I cannot see how I can donate for this amazing project? 

HELP!

**Comment by shymega** [comment]
Hi, thanks for your question. The Barrier project doesn't have a donate feature, as the owner has full control over the project, and the rest of us cannot control the organisation. I have just tried to enable sponsorships for the repo, but it's not working. Until the rest of the team have control over the debauchee organisation/barrier project, there's nothing we can do. But thanks for your kind offer, it's a very good question.

**Comment by shymega** [comment]
Ah! I just figured it out! I have to configure it with a YAML config. I'll email the team and just check how to arrange it.

**Reply by darkdragon-001** [reply]
Any updates?

**Reply by shymega** [reply]
No. I left the Barrier project, as did the other maintainer/developer.

**Reply by darkdragon-001** [reply]
Sad to hear that. So there is no maintainer left? Maybe you should create a pinned issue to look for new maintainers...

**Reply by shymega** [reply]
I can't pin issues. I'm also not interested in Barrier anymore, and have since moved to InputLeap.

**Reply by darkdragon-001** [reply]
Great to hear that the project continues (as [input-leap](https://github.com/input-leap/input-leap) ?). Too bad that you didn't press the _fork_ button in Github to make it discoverable via [_Insichts > Forks_](https://github.com/debauchee/barrier/network/members).

**Reply by shymega** [reply]
We didn't do that to avoid confusion. It was meant to be a fresh start, I _think_. Yes, that is the project.

**Comment by darkdragon-001** [comment]
As a loosely coupled team without legal organization or bank account, I can suggest [OpenCollective](https://opencollective.com/how-it-works) in order to manage the finances transparently :wink:

**Reply by the-spyke** [reply]
Yeah, having at least OpenCollective would be great. The project is not small and I'd like to increase its development speed. I have a Synergy license, and don't mind to pay/contribute for such software if it will start getting rid of bugs actually.

**Reply by shymega** [reply]
As myself and @p12tic have left Barrier, and forked, this discussion would really only be for the remaining members.. that is, if they're active, and I can't make any comments on donations now.

**Reply by tonymynd** [reply]
> I am extremely sorry if this is already somewhere within doc's but I cannot see how I can donate for this amazing project?
> 
> HELP!

Here also, wanted to donate some money, but no obvious way to do it.

**Reply by shymega** [reply]
Barrier is no longer maintained. See 'Input Leap'. We haven't setup funding yet.

**Comment by tonymynd** [comment]
I wonder if there is some legal condition from Seamless corp. to block debauchee/barrier project from receiving donations? I would love to donate money. Can you imagine is all software companies would rely completely on donations? amazing grace.

**Reply by shymega** [reply]
@tonymynd Please don't speculate about things like this. Symless has nothing to do with Barrier's development - Barrier was forked to a new project, and Barrier appears no longer maintained.
